pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
+
+### Build 🧑‍💻
+
+- Jump to our quick [tutorial](./create-your-first-pyth-app/) to get started with Pyth price feeds in your application.
+- Consult our [guides](./use-real-time-data/) and selected the appropriate chain for your application.
+
+**NOTE**: Pyth price feeds are permissionless and do not require a subscription or API key -- developers can complete the whole process in a few minutes.
+
+### Learn 📚
+
+- Check out [How Pyth Works](./how-pyth-works.mdx) to understand the architecture and design of Pyth price feeds.
+- Interact with our on-chain methods at [API-reference](./api-reference.mdx)
```

Comment:
I think the API reference should go in the developer-focused section (?)

It probably doesn't go in the same outline as the two integration guides, but we could have a separate listing of additional developer-focused resources.